src/containers/Footer/Footer.scss

```diff
@@ -0,0 +1,23 @@
+@import '../../styles/variables';
+
+footer {
+  background-color: $darkerBlue !important;
+  min-height: 250px;
+  .container {
```

Comment:
try not to overwrite bootstrap classes, instead make your own class and play with it.

Reply:
do you mean not to use `!important ` to overwrite bootstrap classes ?

Reply:
No, I mean like here you changed the `min-height` by targeting the bootstrap class, I would recommend if you modify a new class for that. for ex:

```
.footerContainer{
    min-height: 250px;
}
```